2653 add event eoi

Calinator444 opened the pull request:
cc: @sethdaily, @camillars 

### Description
This PR adds a re-usable registration of interest form as a Tina component for the SSW Website. The form captures the user's details as well as a link to the event they were interested in. As per my conversation with @sethdaily the details are currently being stored in the JotForms submission data. In addition, a power-automate flow also registers users for our newsletter.

Fixed #2653 


### Note
The power automate flow for adding these users to our contacts hasn't been set up yet. This form will still capture registrations of interest in JotForm but until my flow hit's prod none of the submissions will be added to crm. 

### Screenshots





![ROI button Tina](https://github.com/SSWConsulting/SSW.Website/assets/65635198/cde2dc84-aefe-4304-a4d3-14ce5faf1205)
**Figure**: **Registration of Interest button being edited in Tina**


![Registration of Interest Form](https://github.com/SSWConsulting/SSW.Website/assets/65635198/240f7d21-ca99-4c1c-898a-90430a2f9ea6)
**Figure**: **Registration of interest form**



![image](https://github.com/SSWConsulting/SSW.Website/assets/65635198/1c87bd54-bbd8-4125-a542-811669ae52e8)
**Figure**: **Registration of interest form in Tina**

Calinator444 committed:
💄 Fixed styling on ROI form

Calinator444 committed:
📄 Updated interestForm to include a page URL field.

Calinator444 committed:
🧑‍💻 Made the Registration of interest button text modifiable

Calinator444 committed:
🐛 Fixed live editing for interest form button

Calinator444 committed:
🧪 Performing mimetype test for webp files

Calinator444 committed:
➖ Removed unused "useTina" import

Calinator444 committed:
Revert "🧪 Performing mimetype test for webp files"

Calinator444 committed:
Merge branch 'main' into 2653-add-event-EOI

bradystroud commented on components/events/interestForm.tsx:
```diff
@@ -0,0 +1,33 @@
+  );
+};
+
+const eoiFormId: string = "241638083078865";
```
It would be better to not hardcode this

wicksipedia replied:
agree, should be in global config

Calinator444 replied:
Done - I've moved it to the global config

jeoffreyfischer commented:
The following check has failed - please investigate:
PR - Lint PR/ Check linked issues

Calinator444 commented:
> The following check has failed - please investigate: PR - Lint PR/ Check linked issues

**Done**: I made sure the issue was linked correctly then re-ran the job

Calinator444 committed:
🔐Added expression of interest jot form ID to global settings

Calinator444 committed:
🔧
- Updated interest form to use global setting for registration of interest jotform
- Fixed interest form using formId as page url

Calinator444 committed:
🚨 Fixed build errors

Calinator444 committed:
➖ removed page url from tina schema (the jotform captures it now)

Calinator444 committed:
💾 rebuilt project schema
